File: api/hospital-system/src/main/java/com/api/hospitalsystem/model/role/Role.java
package com.api.hospitalsystem.model.role;

public enum Role {

    Adm("Administrador"),
    User("Usuário"),
    Doctor("Medico"),
    Nurse("Enfermeiro");

    private final String description;

    Role(String description) {
        this.description = description;
    }

    public String getValue() {
        return description;
    }

    @Override
    public String toString() {
        return description;
    }

}

File: app/hospital-system/src/app/module/models/src/role/role.ts
import { describe } from '@module/utils/functions/enum';

export enum Role {
  Adm = 0,
  User = 1,
  Doctor = 2,
  Nurse = 3,
}

describe(Role, {
  Adm: 'Administrador',
  User: 'Usuário',
  Doctor: 'Medico',
  Nurse: 'Enfermeiro',
});
